src/frontend/src/flows/register/finish.ts

```diff
@@ -21,12 +21,14 @@ export const displayUserNumberTemplate = ({
   userNumber,
   identityBackground,
   stepper,
+  slot,
   scrollToTop = false,
 }: {
   onContinue: () => void;
   userNumber: bigint;
   identityBackground: IdentityBackground;
   stepper: TemplateResult;
+  slot?: TemplateResult;
```

Comment:
Can you name it something else?

`slot` seems very generic. The name doesn't need to be about what's inside, maybe about the function that it does inside this template.

I see it's inside a marketing bloc, how about `marketingIntroSlot`?